### ISSUE TITLE
Export

### ISSUE THREAD

**Issue by BruceMoir** [Jack000/Deepnest#132]
Hi Guys,

Having trouble exporting as a .dxf? anyone run into this?
It just exports with no filetype.

**Comment by 9swampy** [comment]
Yes, but if you type the full name including the extension; or change the name of the exported file it should still work fine.
